src/azure-cli/azure/cli/command_modules/security/commands.py

```diff
@@ -261,5 +285,32 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_security_sub_assessments')
         g.custom_command('show', 'get_security_sub_assessment')
 
+    with self.command_group('security iot-solution',
+                            security_iot_solution_sdk,
+                            client_factory=cf_security_iot_solution) as g:
+        g.custom_command('list', 'list_security_iot_solution')
+        g.custom_command('show', 'get_security_iot_solution')
```

Comment:
please use show_command for all show command.

Reply:
I kept it consistent with all the rest of the show commands in the security module 
I don't think we should change only this commands

Reply:
why not from now on? Keep consistent with other CLI commands style is better.

Reply:
show_command will add exception_handler for status code 404, which will be more friendly to customers when error occurs and is consistent with CLI command guideline.

Reply:
changed to show_command